### ISSUE TITLE
[A11y 8/25] Announcer accessibility 

### ISSUE THREAD

**Issue by lpalmaro** [PolymerElements/iron-a11y-announcer#3]
With VoiceOver/Chrome: I can't access the text to the right of the announce buttons. Am I supposed to be able to? After reaching the third announce button, I can then access the "hello my name is ava", but the focus ring appears below the buttons and this doesn't seem logical. Same occurs with CHrome OS / Chromevox. I might just be unclear on how this should work though.

**Comment by goldieep** [comment]
+1

**Comment by dfreedm** [comment]
It looks like the text to the right of the announce buttons is aria hidden. I presume to test that the announce button actually is causing the sentence to be spoken instead of reading the text visible on screen.

**Comment by dfreedm** [comment]
Also, this doesn't seem to actually announce well with VoiceOver, but that is probably a separate bug :(

**Comment by dfreedm** [comment]
This should be fixed after #4, I just have to test on ChromeVox tomorrow before closing.

**Comment by dfreedm** [comment]
ChromeVox works! It has a funky "extra" focus ring around the announcer div though. Maybe @alice knows why...

**Comment by dfreedm** [comment]
I'm going to mark this Fixed and make a separate bug for the focus ring issue.
